### DISCUSSION TITLE
Google IoT Core Support

### DISCUSSION THREAD

**Discussion by WardenCommander** [arendst/Tasmota#13679] [Ideas]
is it possible to connect Tasmota to Google IoT Core? im now it tasmota and would like to control fence gates with esp32 and tasmota on it.

**Comment by sfromis** [comment]
That type of integrations would normally be handled by your backend platform, Home Assistant or whatever

**Reply by WardenCommander** [reply]
Im not planning to use HA at moment, i would like to build my own app. Im just interested is it possible to connect tasmota to Google IoT Core? I saw options where you can build connection from beginning without tasmota.

**Reply by Jason2866** [reply]
Maybe it is similar as it is done with AWS. https://tasmota.github.io/docs/AWS-IoT/

**Reply by WardenCommander** [reply]
I saw that to, will try it but i saw here in closed issues that someone tried to connect with google IoT and there is no answer

**Comment by barbudor** [comment]
I would assume that Google IoT Core, just like AWS IoT, is not just a basic MQTT broker and you have to comply with some topic naming etc... Probably not something that you can do from the basic tasmota configuration.
At least, for AWS, there is a speciifc implementation in Tasmota and I fear the same would be required for Google.

Are you specifically looking into Google IoT or would a generic Cloud based MQTT  broker be enough ?
I've recently started to use Flespi.io for a site where I just have to Tasmota devices and didn't want to go with the burden of setting up neither a local infra or a VPN to my home. Flepsi offer a generic MQTT broker which remains free for up to 50 connected device. You just need to get Tasmota-TLS from https://tasmota.github.io/install/

**Reply by WardenCommander** [reply]
I would prefer Google IoT Core, as im familiar with google services. I will have a look at what you suggest. 
I would like to build more specific implementation in Tasmota if there is some materials/manuals.

**Reply by pawel-smth** [reply]
@WardenCommander , have you managed to get T configured for IoT Core?

**Reply by barbudor** [reply]
Until someone write the code for, that's unlikely to happen

**Comment by geri-m** [comment]
Google IoT Core will be deprecated in August 2023 (https://cloud.google.com/iot/docs/release-notes), so I guess this discussion can be closed.

**Reply by sfromis** [reply]
Discussions are not closed, but just hibernate unless someone wakes them up
